prime-dotnet-webapi/ViewModels/SiteRegistration/ClaimOrganizationViewModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace Prime.ViewModels
+{
+    public class ClaimOrganizationViewModel
```

Comment:
Since "claim" reads more as a noun than a verb, this reads better as `OrganizationClaimViewModel`